api/wikis/views.py

```diff
@@ -131,6 +143,15 @@ def get_serializer_class(self):
     def get_object(self):
         return self.get_wiki()
 
+    # overrides RetrieveUpdateDestroyAPIView
+    def perform_destroy(self, instance):
+        auth = get_user_auth(self.request)
+        wiki_page = self.get_object()
+        try:
+            wiki_page.node.delete_node_wiki(wiki_page, auth)
```

Comment:
Drive-by: Perhaps we should refactor to put the `delete...` method on `wiki_page` instead of node (see [Law of Demeter](https://www.wikiwand.com/en/Law_of_Demeter)).